segmented_sieve_of_eratosthenes/segmentedSieveOfEratosthenes.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+#include <cmath>
+#include <cstring>
+#include <vector>
+using namespace std;
+
+vector<int> Prime; //contains prime numbers uptop segMax
```

Comment:
Please do not use global variables.

Reply:
This is so that one can directly use Prime vector to find get all prime numbers just after calling segmentedSieve()..